tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_pipeline_job.py

```diff
@@ -17,7 +17,7 @@
 # under the License.
 
 # mypy ignore arg types (for templated fields)
-# type: ignore[arg-type]
+
 
```

Comment:
I think the “mypy ignore” comment above should be removed as well

Reply:
Same for multiple other files

Reply:
Correct. I reviewed it everywhere

Reply:
I reviewed it and made consistent everywhere (removed old comment if not needed and explained where it is added to avoid missing default args warning. BTW. Good that we have those` # mypy: disable-error-codes` on a global level.